application/database/database_routes.py

```diff
@@ -20,15 +20,20 @@ def s3_upload(file_path, bucket_name):
   response = s3.upload_file(file_path, bucket_name, f'unprocessed/{file_path}')
   return response
 
-def str_to_bool(s):
+def any_to_bool(s):
   """
-  If s is none it will pass so the default value in the model is used, otherwise will return true 
-  if one of those strings matches
+  If s can't be converted to a bool it will pass so the default value in the model is used,
+  otherwise will return true if it's truthy
   """
-  if s is None:
-    pass
-  else:
-    return s.lower() in ['true', 'TRUE', 'True', '1', 'yes', 'y', 't']
+  if type(s) == bool:
+    return s
+  elif type(s) == int:
+    return s == 1
+  elif type(s) == str:
+    return s.lower() in ['true', '1', 'yes', 'y', 't']
```

Comment:
Should the `1` still be in here? If not, prob best to take out

Reply:
yes because `'1'` is still a string, `1` is not